jupyter-extension/src/plugins/mount/components/Pipeline/hooks/usePipeline.tsx

```diff
@@ -47,7 +47,7 @@ export const usePipeline = (
   const [pipelineProject, setPipelineProject] = useState('');
   const [imageName, setImageName] = useState('');
   const [inputSpec, setInputSpec] = useState('');
-  const [pipelinePort, setPipelinePort] = useState('');
+  const [pipelinePort, setPipelinePort] = useState(1234);
```

Comment:
I unfortunately think this should remain a string integer than can be the empty string, unless further changes are made. If this value is an integer, there's no way to "unset" it. The backend will then always create the pipeline as a service pipeline, which is not what we want, as service pipelines have no output repo.

Its clear that this `port` field probably needs a rework, so in my opinion we should file a ticket for that and revert these changes.